packages/databricks-vscode/src/bundle/run/JobRunStatus.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    JobsError,
+    JobsRetriableError,
+} from "@databricks/databricks-sdk/dist/apis/jobs";
+import {Event, EventEmitter} from "vscode";
+import {AuthProvider} from "../../configuration/auth/AuthProvider";
+import {onError} from "../../utils/onErrorDecorator";
+import {BundleRunStatus} from "./BundleRunStatus";
+
+export class JobRunStatus extends BundleRunStatus {
+    readonly type = "jobs";
+
+    private readonly onDidCompleteEmitter = new EventEmitter<void>();
+    onDidComplete: Event<void> = this.onDidCompleteEmitter.event;
+
+    constructor(private readonly authProvider: AuthProvider) {
+        super();
+    }
+
+    parseId(output: string) {
+        if (this.runId !== undefined || this.runState !== "unknown") {
+            return;
+        }
+        const match = output.match(/.*\/run\/(\d*).*/);
+        if (match === null) {
+            return;
+        }
+        this.runId = match[1];
+
+        this.startPolling();
+    }
+
+    @onError({popup: {prefix: "Error starting polling."}})
```

Comment:
For a user facing popup more descriptive name would be better - something like "Failed to check the run status of the job". Similar comment for pipelines